### ISSUE TITLE
Cannot enable CAN for FMU-V5 - is 'fmu-v5/src/can.c' wrong?

### ISSUE THREAD

**Issue by BenAdamsonWecorp** [PX4/Firmware#16058]
I am trying to enable CAN on my CUAV V5+ for a custom application (not UAVCAN).

From reading [fmu-v5/src/can.c](https://github.com/PX4/Firmware/blob/master/boards/px4/fmu-v5/src/can.c), I believe to enable this driver I need to use ‘menuconfig’ to enable CONFIG_STM32_CAN1 among others. Unfortunately I cannot find a way to enable this (and it seems the nuttx-config/nsh/defconfig file renames this to CONFIG_STM32F7_CAN1 for this board.

Should I edit the source code of fmu-v5/src/can.c to use CONFIG_STM32F7_CAN1 in place of CONFIG_STM32_CAN1, or am I missing something here?

**Comment by dagar** [comment]
> Should I edit the source code of fmu-v5/src/can.c to use CONFIG_STM32F7_CAN1 in place of CONFIG_STM32_CAN1, or am I missing something here?

Yes you can edit that to work like the can init in actual NuttX boards. We should probably add something simple to the builds like `px4_fmu-v5_can` that has NuttX CAN enabled and skips UAVCAN.
